examples/pvtol-lqr-nested.ipynb

```diff
@@ -57,21 +74,9 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
-   "metadata": {},
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "m = 4.000000\n",
-      "J = 0.047500\n",
-      "r = 0.250000\n",
-      "g = 9.800000\n",
-      "c = 0.050000\n"
-     ]
-    }
-   ],
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
```

Comment:
You should xecute the notebook, so that the checked in version can be viewed completely on github.

Compare
https://github.com/python-control/python-control/blob/master/examples/pvtol-lqr-nested.ipynb

to

https://github.com/python-control/python-control/blob/2d302ba23919f0479ccbcbb0ea85523489a1e27a/examples/pvtol-lqr-nested.ipynb